src/main/java/core/mate/academy/service/MachineServiceImpl.java

```diff
@@ -1,7 +1,40 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Excavator;
+import core.mate.academy.model.Machine;
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * Your implementation of MachineService.
  */
-public class MachineServiceImpl {
+public class MachineServiceImpl implements MachineService<Machine> {
+    @Override
+    public List<? extends Machine> getAll(Class type) {
+        if (Bulldozer.class == type) {
+            BulldozerProducer bulldozerProducer = new BulldozerProducer();
+            return bulldozerProducer.get();
```

Comment:
Return as indicated in the checklist
<img width="1016" alt="Screen Shot 2022-10-08 at 3 41 37 PM" src="https://user-images.githubusercontent.com/90692308/194708138-67b08935-9954-4314-8461-660e884b254e.png">